### PR TITLE
[php] #126: Not-required properties now shows as nullable

### PR DESCRIPTION
**Pull request by fMads**
### PR checklist

- [X] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [X] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [X] Filed the PR against the correct branch: Default: `master`.
- [X] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Fixed issue #126 

### Technical Committee
- @jebentier
- @dkarlovi
- @mandrean
- @jfastnacht
- @ackintosh

### CODE REVIEW

**Comment by jebentier** [comment]
It looks like there are some tests that are failing due to the nullable failures.  Please look into these and fix them as needed.

File: samples/client/petstore/php/OpenAPIClient-php/docs/Model/Capitalization.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**small_camel** | **string** |  | [optional] 
-**capital_camel** | **string** |  | [optional] 
-**small_snake** | **string** |  | [optional] 
-**capital_snake** | **string** |  | [optional] 
-**sca_eth_flow_points** | **string** |  | [optional] 
-**att_name** | **string** | Name of the pet | [optional] 
+**small_camel** | **object** |  | [optional]
```
Comment (wing328) [inline samples/client/petstore/php/OpenAPIClient-php/docs/Model/Capitalization.md]: @fMads looks like something wrong. Not related to change in this PR. All type becomes "object".

Can you rebase this PR on the latest master (or merge the latest master into this PR), build the JAR (mvn clean package) and try to update the PHP samples again?

Comment (fMads) [reply samples/client/petstore/php/OpenAPIClient-php/docs/Model/Capitalization.md]: Yes - done :)

**Comment by wing328** [comment]
@fMads FYI. I've updated the PHP samples via https://github.com/OpenAPITools/openapi-generator/pull/169

**Comment by fMads** [comment]
@wing328 Thanks :)

**Pull request merged**